src/Cassandra.IntegrationTests/Core/TransitionalAuthenticationTests.cs

```diff
@@ -0,0 +1,140 @@
+//
+//      Copyright (C) DataStax Inc.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+using System.Diagnostics;
+using System.Linq;
+
+using Cassandra.DataStax.Auth;
+using Cassandra.IntegrationTests.TestBase;
+using Cassandra.IntegrationTests.TestClusterManagement;
+using Cassandra.Tests;
+
+using NUnit.Framework;
+
+namespace Cassandra.IntegrationTests.Core
+{
+    /// <summary>
+    /// Test Cassandra Authentication.
+    /// </summary>
+    [TestFixture, Category(TestCategory.Short), Category(TestCategory.RealCluster), Category(TestCategory.ServerApi)]
+    [TestDseVersion(5, 0)]
+    public class TransitionalAuthenticationTests : TestGlobals
+    {
+        // Test cluster object to be shared by tests in this class only
+        private ITestCluster _testClusterForAuthTesting;
+
+        public void RetryUntilClusterAuthHealthy(ITestCluster cluster)
+        {
+            using (var c = ClusterBuilder()
+                           .AddContactPoint(cluster.InitialContactPoint)
+                           .WithAuthProvider(new PlainTextAuthProvider("wrong_username", "password"))
+                           .WithSocketOptions(new SocketOptions().SetReadTimeoutMillis(22000).SetConnectTimeoutMillis(60000))
+                           .Build())
+            {
+                TestHelper.RetryAssert(
+                    () =>
+                    {
+                        using (var _ = c.Connect())
+                        {
+                        }
+                    },
+                    500,
+                    300);
+            }
+        }
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            Diagnostics.CassandraTraceSwitch.Level = TraceLevel.Info;
+            _testClusterForAuthTesting = GetTestCcmClusterForAuthTests();
+            RetryUntilClusterAuthHealthy(_testClusterForAuthTesting);
```

Comment:
Is this related to a failure? I didn't get it on the Node.js driver with `-Dcassandra.superuser_setup_delay_ms=0`.

Reply:
Tbh I'm not sure why this is here, I copied this from the pre exiting authentication tests... I can give it a go and just remove this

Reply:
bah, whatever you prefer, I think this was needed before "delay" setting to 0ms but its not anymore.

Reply:
It seems that it wasn't needed indeed, tests are green without it :+1: